sites/public/pages/eligibility/accessibility.tsx

```diff
@@ -0,0 +1,235 @@
+/*
+Accessibility
+Prompts the user for their age to filter for properties that are age dependent.
+*/
+import {
+    AppearanceStyleType,
+    Button,
+    FormCard,
+    t,
+    Form,
+    ProgressNav,
+    FieldGroup,
+    Field,
+  } from "@bloom-housing/ui-components"
+  import FormsLayout from "../../layouts/forms"
```

Comment:
Are these spaces at the beginning of the lines right?

Reply:
fixed